lib/sof/runner.rb

```diff
@@ -0,0 +1,35 @@
+require 'sof/check'
+require 'sof/server'
+require 'sof/checks'
+
+module Sof
+class Runner
+
+  attr_accessor :server_concurrency, :check_concurrency
+
+  def initialize
+    server_concurrency ||= 0
```

Comment:
0 disables the use of threads/processes.  our real defaults will be 10 and 5 I think.